l1-contracts/src/core/libraries/ConstantsGen.sol

```diff
@@ -31,25 +31,7 @@ library Constants {
   uint256 internal constant MAX_NOTE_ENCRYPTED_LOGS_PER_CALL = 16;
   uint256 internal constant MAX_ENCRYPTED_LOGS_PER_CALL = 4;
   uint256 internal constant MAX_UNENCRYPTED_LOGS_PER_CALL = 4;
```

Comment:
I had to re-order the constants here so that they get successfully evaluated in TS with the eval method --> the constants became interdependent